cmd/ddltest/main_test.go

```diff
@@ -37,6 +37,8 @@ func TestMain(m *testing.M) {
 		goleak.IgnoreTopFunction("go.opencensus.io/stats/view.(*worker).start"),
 		goleak.IgnoreTopFunction("internal/poll.runtime_pollWait"),
 		goleak.IgnoreTopFunction("net/http.(*persistConn).writeLoop"),
+		goleak.IgnoreTopFunction("github.com/go-sql-driver/mysql.(*mysqlConn).startWatcher.func1"),
```

Comment:
Why do we ignore the remained connection goroutine?

IMO if we need to close the `db` created: https://github.com/pingcap/tidb/blob/e97e8c0f238387c33cccca23f2434389f5f39c10/cmd/ddltest/ddl_serial_test.go#L290

Reply:
It's already closed here: https://github.com/pingcap/tidb/blob/7d4895fa96b7b6fade76b995d8a4d572768cd2f7/cmd/ddltest/ddl_serial_test.go#L244-L247

But sometime sometimes it still have some goroutine leaks. I guess maybe some gorutine is slow to exist. However, ignoring mysql client goroutine is safe in this case

Reply:
Got it, thanks.